### DISCUSSION TITLE
Flow launcher doesnt launch item on monitor where search window is

### DISCUSSION THREAD

**Discussion by mx3214** [Flow-Launcher/Flow.Launcher#2351] [Q&A]
I work on a dual monitor set up.

I like that I can set the 'search window' position to the monitor with my cursor on it.

However, I find that even if I'm working on monitor 1, and then move my mouse to monitor 2, open flow launcher (and the 'search window' will open on monitor 2 as that's where my mouse is), it still launches a selected item on monitor 1. 

Is it possible to get flow launcher to launch a selected item on the monitor where the 'search window' is present?

Sorry I know that's the most clear thing to follow, but hope it makes some sense.

**Comment by mx3214** [comment]
I don't know even know if clicking on monitor 2 will ensure the item will open on monitor 2.

Any information on why an item is opened in a specific monitor on a dual monitor display by flow launcher would be very much appreciated.

**Reply by jjw24** [reply]
Hey, are you opening a bookmark/website?

**Reply by mx3214** [reply]
I'm opening various things; mainly urls on edge, word documents and folders (in windows 11)

**Reply by jjw24** [reply]
Pretty sure urls and bookmarks, if you have open as a tab, Windows will pick a monitor with a browser already open to open the new tab if your current one does not have a browser open. Or is this behaviour happening even for open urls/bookmarks in new window after clicking on the desired screen?

**Reply by mx3214** [reply]
I've got flow launcher set so that it opens urls in a new window, not in a tab. But yes if I had it set to tab, then it would open wherever the browser window is open

**Reply by jjw24** [reply]
Will have to test later to see what's going on.
